libsession/src/main/java/org/session/libsession/snode/SnodeAPI.kt

```diff
@@ -206,6 +226,10 @@ object SnodeAPI {
         }
     }
 
+    private fun extractVersionString(jsonVersion: String): String{
+        return jsonVersion.removeSurrounding("[", "]").split(", ").joinToString(separator = ".")
```

Comment:
Nice! I did not know `removeSurrouding` exists, I'd almost grab regex to start with